packages/lab/stories/slider/slider.qa.stories.tsx

```diff
@@ -3,94 +3,65 @@ import type { Meta, StoryFn } from "@storybook/react";
 import { QAContainer, type QAContainerProps } from "docs/components";
 
 export default {
-  title: "Lab/Slider/Slider QA",
```

Comment:
This should be changed back, it makes it easier to tell what story you're dealing with on Chromatic